01_01_Basic_Rules.txt

```diff
@@ -0,0 +1,1311 @@
+fire with +1 miss chance, not-stackable. Suppression status can't be purged 
+unless you move outside the AOE. The only other way to stop being suppressed 
+is to stop the suppressor from using a suppression action on you at the end 
+of their turn.
+
+Moving out into the open: 
+	In the context of suppression and movement, "moving out into the open" 
+includes moving into a new location while you can be hit by shots from the 
+suppressor (this excludes moving into complete concealment, such as going prone 
+behind a high concrete wall). For example, the following situations would 
+trigger a free hit by the suppressor and give you the 'suppressed' status while 
+remaining in the area (let's assume you are upright (not crouched or prone)):
+moving from partial cover to an adjacent partial cover, moving from complete 
+concealment behind a barricade to partial cover, and moving when you are caught 
+out in the open.
+
+	If the suppressor cannot physically hit you (such as being completely behind 
+an impenetrable barrier) you are not suppressed. The suppressor doesn't need to 
+see you to hit you, as in the case of suppressing a cloaked target. 
+
+Light Suppression:
+	As the last action of your turn, aim at a target and fire 3 rounds (or the 
+rest of your magazine, at least 1 round has to be fired) to suppress 1m radius 
+AOE around the target. The target takes a +1 miss chance malus until the start 
+of your next turn, and if the target moves out into the open**, then roll an 
+immediate reaction to hit the target with a (non-automatic fire) accuracy roll. 
+Immediate reactions are stackable on a per-weapon basis.
+
+Medium/Moderate Suppression:
+	identical to light suppression with the following exceptions, you must fire 
+6 rounds from the current magazine to achieve 'moderate', and you don't have to 
+roll to hit -- you automatically hit and roll armor.
+
+Heavy Suppression:
+	identical to light suppression with the following exceptions: you must fire 
+10 rounds from the current magazine to achieve 'heavy', you don't have to roll 
+to hit -- you automatically hit x2 and roll for armor. You suppress all targets 
+within a 2m radius AOE around the primary target.
+
+
+**"Moving out into the open" includes moving into a new location while you can
+be hit by shots from the suppressor (this excludes moving into complete concealment,
+such as going prone behind a high concrete wall). For example, the following situations
+would trigger a free hit by the suppressor and give you the 'suppressed' status
+while remaining in the area (let's assume you are upright (not crouched or prone)):
+moving from partial cover to an adjacent partial cover, moving from complete concealment
+behind a barricade to partial cover, and moving when you are caught out in the open.
+	If the suppressor cannot physically hit you (such as being completely behind an 
+impenetrable barrier) you are not suppressed. The suppressor doesn't need to see 
+you, to hit you.

```

Comment:
These two paragraphs the way I originally wrote suppression. @bleehu after our phone conversation, I thought we were going to keep the footnotes? 
If that is the case, the two paragraphs after line 1098 need to be removed, and the asterisks that denote the reference to this footnote, need to be restored. I'm not sure what happened to the first footnote, because I wouldn't have used two asterisks unless there were two footnotes.

Reply:
I did keep the footnotes? I modified the parser specifically to allow for multi-paragraph footnotes so we could keep these. 

As best I can tell from digging through the git history, there was a single one and then you changed it to two?: https://github.com/bleehu/Compound_X/commit/fc2e400052c2386c9e728cf63f10511411d0739d

Reply:
I wonder if part of my post got cut-off somehow. Or I miswrote.
What I should've said was, there are two instances of the same writing; one as the original footnotes, and another as paragraphs. Lines 1096-1109 are the paragraphs.
https://github.com/bleehu/Compound_X/blob/2af435f3bf1976ba42eb9eb8b310104b2dde0a97/01_01_Basic_Rules.txt#L1096

Ahh, sorry for misplacing the blame (heh). Idk why I put those two asterisks in. Thanks for looking that up! And thanks for your hard work on the parser!

Reply:
--blocking